training/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,29 @@
+Cloud Services and Infrastructure in Digital Publishing
+===========================
+
+We use Amazon Web Services in Digital Publishing to host all of our services, as well as our Continuous Integration (CI).
+
+You should be given relevant AWS accounts by your manager or a Tech Lead - speak to your manager if this does not happen soon upon starting with the team. 
+
+
+Introduction to our Infrastructure
+----------------------------
+
+In your first few weeks you should be invited to a number of 'Intro Workshops', one of which will be on the topic of Infrastructure. 
```

Comment:
I wonder if the part about "in the first few weeks you will..." perhaps fits better in the top level readme with more of a synopsis of what this workshop will cover here instead?